### PR TITLE
chore: update documentation links

### PR DESCRIPTION
**Pull request by ashin-czi**
#2784

#### Reviewers
**Functional:** @seve 

---

## Changes
- add
- remove
- modify
Updating URL strings

### CODE REVIEW

File: client/src/components/floatingButton/floatingButton.tsx [hunk 1]
```diff
@@ -24,23 +24,23 @@ function FloatingButton(): JSX.Element {
...
             rel="noopener"
             target="_blank"
             text="Documentation"
           />
           <MenuItem
-            href="https://github.com/chanzuckerberg/cellxgene-documentation/blob/main/explore-data/explorer-tutorials.md"
+            href="https://cellxgene.dev.single-cell.czi.technology/docs/04__Analyze%20Public%20Data/4_1__Hosted%20Tutorials"
             rel="noopener"
             target="_blank"
             text="Tutorials"
```
Comment (ashin-czi) [inline client/src/components/floatingButton/floatingButton.tsx:36]:
```suggestion
            target="_blank"
            text="Tutorials"
```

File: client/src/components/floatingButton/floatingButton.tsx [hunk 2]
```diff
@@ -24,23 +24,23 @@ function FloatingButton(): JSX.Element {
       content={
         <Menu>
           <MenuItem
-            href="https://github.com/chanzuckerberg/cellxgene-documentation/blob/main/README.md"
+            href="https://cellxgene.dev.single-cell.czi.technology/docs"
```
Comment (seve) [inline client/src/components/floatingButton/floatingButton.tsx:27]: This url is hardcoded to dev, we'll want this url to change as we go from deployment to deployment. So it's probably best to make it dynamic based on the data portal home page link passed here: https://github.com/chanzuckerberg/single-cell-explorer/blob/1b10f5ed62715ee3ecce1a352765c3a069f4db5d/client/src/globals.ts#L31-L47

File: client/src/components/floatingButton/floatingButton.tsx [hunk 3]
```diff
@@ -24,23 +24,23 @@ function FloatingButton(): JSX.Element {
...
             text="Tutorials"
           />
-          <MenuItem
+          {/* <MenuItem
             href="https://github.com/chanzuckerberg/cellxgene-documentation/blob/main/roadmap.md"
             rel="noopener"
             target="_blank"
             text="Our Roadmap"
-          />
+          /> */}
```
Comment (seve) [inline client/src/components/floatingButton/floatingButton.tsx:43]: Let's just delete this if we aren't rendering it anymore

**Review by seve — CHANGES_REQUESTED** [review]

File: client/src/components/floatingButton/floatingButton.tsx (hunk 2 above)
Comment (ashin-czi) [reply client/src/components/floatingButton/floatingButton.tsx]: Just pushed some changes for this

**Review by seve — APPROVED** [review]

**Pull request merged**